xml2xls/xls2xml.py

```
@@ -132,6 +132,7 @@ def start_convert(options):
 def main():
     options = add_parser()
     start_convert(options)
+    bbbbbbbbbbb
```

Ruff (0.13.3)

Found useless expression. Either assign it to a variable or remove it.

(B018)